uk.co.compucorp.civicrm.contactactionsmenu/contactactionsmenu.civix.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
+// AUTO-GENERATED FILE -- Civix may overwrite any changes made to this file
+
+/**
+ * The ExtensionUtil class provides small stubs for accessing resources of this
+ * extension.
+ */
+class CRM_Contactactionsmenu_ExtensionUtil {
+  const SHORT_NAME = "contactactionsmenu";
+  const LONG_NAME = "uk.co.compucorp.civicrm.contactactionsmenu";
+  const CLASS_PREFIX = "CRM_Contactactionsmenu";
```

Comment:
this will need to be updated too

Reply:
I wanted to change this but since its an autogenerated file I was wondering if it won't be reverted back?

Reply:
I don't think it will. This file is autogenerated but it's never updated as far as I concerned, so the changes won't be reverted